PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/actions/WebDeployAction.java

```diff
@@ -62,50 +63,38 @@ public void actionPerformed(AnActionEvent event) {
             return;
         }
 
-        ApplicationManager.getApplication().invokeLater(() -> {
-            runConfiguration(project);
-        });
+        ApplicationManager.getApplication().invokeLater(() -> runConfiguration(project));
     }
 
     private void runConfiguration(Project project) {
-        final RunManagerImpl manager = RunManagerImpl.getInstanceImpl(project);
```

Comment:
Sounds like a great feature!

So, that's the reason to seperate our changes into small commits for review. Good for reviewers, also good for coders. Fg, if the change needs to be reverted, it won't effect your new feature.